src/main/resources/SLING-INF/content/static

```diff
@@ -0,0 +1 @@
+../../../../../../rhdocs-styles/stylesheets
```

Comment:
Is this a symlink to files outside of the project? If so, we might need a better way to do this.

Reply:
It sure is. I'm open to suggestions. I had briefly discussed the idea of a git submodule with Ivan but he was pretty opposed, calling it more trouble than it was worth.

The templates & stylesheets are stored in Mark's personal repository for right now... how should we handle that?

Reply:
Yeah, git submodules are a bad idea imo too. I'll resolve the discussion for now, but we need to figure out how to manage these types of dependencies.

We've used webjars (https://www.webjars.org) in the past, so we could look at it.

@markcaron any ideas?